### ISSUE TITLE
sail-to-lean: iterate over imports

### ISSUE THREAD

**Issue by tobiasgrosser** [rems-project/sail#839]

**Comment by tobiasgrosser** [comment]
@javra, what is the status here? Did we decide not to pursue this at the moment?

**Comment by javra** [comment]
Yes, we shelved it under "cosmetic improvements to be done after functional things"

**Comment by tobiasgrosser** [comment]
This has been superseeded by https://github.com/rems-project/sail/issues/1290.
